packages/web/src/common/store/backend/sagas.ts

```diff
@@ -68,6 +61,11 @@ export function* setupBackend() {
   // Optimistically fetch account, then do it again later when we're sure we're connected
   // This ensures we always get the cached account when starting offline if available
   yield* put(accountActions.fetchLocalAccount())
+
+  // Init APICLient
+  const apiClient = yield* getContext('apiClient')
```

Comment:
do we need to move this now that we are technically just going through with init?

Reply:
I think the change is not required, but it works this way too